### PR TITLE
fix permissions for specified files in DotNetCoreInstallerV0, DotNetCoreInstallerV1, UseDotNetV2 tasks

### PR DESCRIPTION
**Pull request by PavloAndriiesh**
**Task name**: DotNetCoreInstallerV0, DotNetCoreInstallerV1, UseDotNetV2 

**Description**: Set safe permissions 755 for files in tasks to prevent local privilege escalation on Linux

**Documentation changes required:** (Y/N) <Please mark if documentation changes are required>

**Added unit tests:** (Y/N) <Please mark if unit tests were added or updated according changes>

**Attached related issue:** (Y/N) [MSRC 70231 - AzureDevOps - ElevationOfPrivilege - Local Privilege Escalation - Microsoft Azure Pipelines Tasks on Linux](https://dev.azure.com/mseng/AzureDevOps/_workitems/edit/1927995)

**Checklist**:
- [+] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [+] Checked that applied changes work as expected

### CODE REVIEW

File: Tasks/DotNetCoreInstallerV0/task.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 0,
-        "Minor": 198,
+        "Minor": 199,
```
Comment (rohit-batra) [inline Tasks/DotNetCoreInstallerV0/task.json:16]: I guess this is outdated (We usually keep this as Sprint number)

File: Tasks/DotNetCoreInstallerV1/task.json [hunk 2]
```diff
@@ -13,7 +13,7 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 1,
-        "Minor": 198,
+        "Minor": 199,
```
Comment (rohit-batra) [inline Tasks/DotNetCoreInstallerV1/task.json:16]: This too

File: Tasks/UseDotNetV2/task.json [hunk 3]
```diff
@@ -13,8 +13,8 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 2,
-        "Minor": 198,
-        "Patch": 1
+        "Minor": 200,
```
Comment (rohit-batra) [inline Tasks/UseDotNetV2/task.json:16]: here and in all such places. Also consistency issue ()Here its 200 in other places 199)

**Review by rohit-batra — APPROVED** [review]
Some nits, please address them before committing.

File: Tasks/DotNetCoreInstallerV1/task.json (hunk 2 above)
Comment (PavloAndriiesh) [reply Tasks/DotNetCoreInstallerV1/task.json]: fixed with 204

File: Tasks/UseDotNetV2/task.json (hunk 3 above)
Comment (PavloAndriiesh) [reply Tasks/UseDotNetV2/task.json]: fixed with 204

**Review by rohit-batra — CHANGES_REQUESTED** [review]
You will have to build this as well so that corresponding task.loc.json gets generated with right Minor version and then add those task.loc.json (unless process has changed in last couple of years)

**Review by rohit-batra — APPROVED** [review]

**Comment by mmrazik** [comment]
This test won't pass: https://github.com/microsoft/azure-pipelines-tasks/blob/acc64cc7292c98597908325e53af9f898a896189/Tasks/DotNetCoreInstallerV0/Tests/L0.ts#L142

**Pull request merged**